### PR TITLE
DOC: Changes to revert partial credit doc updates

### PR DESCRIPTION
**Pull request by catong**
This PR reverts partial credit documentation updates until the feature is re-merged into master.

@mhoeber or @lamagnifica could you please give this a check? I compared against https://github.com/edx/edx-documentation/pull/513 to revert changes. I did not delete the images that were added in PR #513, since if they're not referenced they should be fine. PR #513 included some style guide type edits, which I also left in. 
@mhoeber There seemed to be some reorganization in custom_python.rst that I didn't have context for, so please check that one in particular.
FYI @explorerleslie

### CODE REVIEW

File: en_us/shared/exercises_tools/checkbox.rst [hunk 1]
```diff
@@ -14,17 +14,13 @@ Overview
...
 
 .. image:: ../../../shared/building_and_running_chapters/Images/CheckboxExample.png
- :alt: A checkbox problem with four options, 2 of which are correct.
+ :alt: A checkbox problem with four options, 2 of which are required for the correct answer.
 
-.. note:: 
-   Make sure that all of the answer choices are unambiguous, and avoid trick
-   questions. Checkbox problems with ambiguity can be frustrating to learners,
-   especially if they have a limited number of attempts.
+.. commenting out for partial credit revert.. note:: Make sure that all of the answer choices are unambiguous, and avoid trick questions. Checkbox problems with ambiguity can be frustrating to learners, especially if they have a limited number of attempts.
```
Comment (lamagnifica) [inline en_us/shared/exercises_tools/checkbox.rst:26]: @catong this note moved from / to lines 33-37 in this PR. I don't think the note is necessary here.

**Comment by lamagnifica** [comment]
one comment, ignore if desired. :+1:

File: en_us/shared/exercises_tools/checkbox.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/exercises_tools/checkbox.rst:26]: @lamagnifica Yep, I restored the original, but I think the note is better where it was. Will uncomment the note and remove it from its old position.

**Comment by catong** [comment]
@mhoeber Do you want to give thumbs before I merge this PR?

**Comment by mhoeber** [comment]
:+1:   thanks

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/533

**Pull request merged**